packages/rx_bloc_generator/pubspec.yaml

```diff
@@ -16,7 +16,7 @@ dependencies:
   dart_style: ">=1.3.0 <3.0.0"
   logging: ^1.0.2
   rx_bloc: ^4.0.0
-  source_gen: ">=0.9.10 <2.0.0"
+  source_gen: 1.2.7
```

Comment:
Why the version needs to be exact ?

Reply:
On one of my previous commits on this branch, the tests started failing and as something different I found that the version of source_gen  that was installed was the newest 1.3.0, which is from 4 days ago, here is this commit:
https://github.com/Prime-Holding/rx_bloc/actions/runs/4892931582/jobs/8735276862

And I found another commit from 2 weeks ago, when the tests were not failing where the installed version of source_gen was 1.2.7 : 
https://github.com/Prime-Holding/rx_bloc/actions/runs/4831772148/jobs/8609653384, so I set the version with which the tests are not failing. I didn't find another reason related to the error, why the tests are failing.

Now I see that there is a new of version source_gen_test 1.0.5 from 3 days ago, which is installed with ,source_gen 1.3.0, the tests are not failing any more, so I have returned the range dependency for source_gen.